db/db_impl.cc

```diff
@@ -4000,18 +4000,30 @@ Status DBImpl::GetImpl(const ReadOptions& read_options,
   auto cfh = reinterpret_cast<ColumnFamilyHandleImpl*>(column_family);
   auto cfd = cfh->cfd();
 
+  // Acquire SuperVersion
+  SuperVersion* sv = GetAndRefSuperVersion(cfd);
+
+  TEST_SYNC_POINT("DBImpl::GetImpl:1");
```

Comment:
The test case is generally not valid after moving the place we get snapshot. If you think that will be a good test case too, I can create another test case to cover what you mentioned.